examples/collada.html

```diff
@@ -0,0 +1,53 @@
+<html>
+    <head>
+        <title>Itowns - collada</title>
+
+        <style type="text/css">
+            html {
+                height: 100%;
+            }
+
+            body {
+                margin: 0;
+                overflow:hidden;
+                height:100%;
+            }
+
+            #viewerDiv {
+                margin : auto auto;
+                width: 100%;
+                height: 100%;
+                padding: 0;
+            }
+
+            #menuDiv {
+                position: absolute;
+                top:0px;
+                margin-left: 0px;
+            }
+
+            @media (max-width: 600px) {
+                #menuDiv {
+                    display: none;
+                }
+            }
+        </style>
+        <meta charset="UTF-8">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="GUI/dat.gui/dat.gui.min.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv"></div>
+        <script src="GUI/GuiTools.js"></script>
+        <script src="../dist/itowns.js"></script>
+        <script src="../dist/debug.js"></script>
+        <script type="text/javascript">
+            var renderer;
+            var exports = {};
+            var THREE = itowns.THREE;
+        </script>
+        <script src="https://cdn.rawgit.com/mrdoob/three.js/dev/examples/js/loaders/ColladaLoader.js"></script>
```

Comment:
Oh forgot this: let's point to our current tag version: v0.87 (we'll need to remember changing that when we update three.js version)

Reply:
And I need to think about a way to use them in our build without this trick...

Reply:
yes we have to find a solution
Here, we point to dev THREE version

Reply:
>  Here, we point to dev THREE version

yes, that's my point, let's point to the tag here instead to ensure compatibility with our npm version.

Reply:
i updated to check npm version